docs/API.md

```diff
@@ -171,7 +176,17 @@ curl -v \
 ```json
 {
   "id": "81625c14128d46c2b600e74a017fa4a8",
-  "url": "https://secure.gravatar.com/avatar/6d940dd41e636cc156074109b8092f96"
+  "avatar": "https://secure.gravatar.com/avatar/6d940dd41e636cc156074109b8092f96"
+}
+```
+
+#### Response (no avatar set)
+
+```json
+{
+  "default": true,
```

Comment:
would this be useful?

Reply:
On reflection, I think it's fine to just be `default` here because the URL is specifically asking for information about the avatar.  I guess this key would simply not show up in the response for the combined `/profile` endpoint?